### PR TITLE
feat: did-exchange REST endpoints to use controller

### PR DESCRIPTION
**Pull request by sudeshrshetty**
- DIDExchange REST operations to use controller commands
- Removed `pkg/restapi` and all restapi initialization and opts moved to
`pkg/controller/restapi`

Signed-off-by: sudesh.shetty <sudesh.shetty@securekey.com>

### CODE REVIEW

File: pkg/controller/restapi/operation/messaging/operation.go [hunk 1]
```diff
@@ -13,14 +13,15 @@ import (
 	"fmt"
 	"net/http"
 
+	"github.com/hyperledger/aries-framework-go/pkg/controller/restapi/internal/exec"
+
```
Comment (rolsonquadras) [inline pkg/controller/restapi/operation/messaging/operation.go]: extra line.

File: pkg/controller/restapi/operation/vdri/operation.go [hunk 2]
```diff
@@ -15,11 +15,13 @@ import (
 	"net/http"
 	"net/url"
 
+	"github.com/hyperledger/aries-framework-go/pkg/controller/restapi/internal/exec"
+
```
Comment (rolsonquadras) [inline pkg/controller/restapi/operation/vdri/operation.go]: extra line.

File: pkg/controller/restapi/restapi_test.go [hunk 3]
```diff
@@ -1,8 +1,11 @@
 /*
-Copyright SecureKey Technologies Inc. All Rights Reserved.
```
Comment (rolsonquadras) [inline pkg/controller/restapi/restapi_test.go]: any reason to change the format ? Most of the files use this format.

Comment (sudeshrshetty) [reply pkg/controller/restapi/restapi_test.go]: looks like my IDE is autofixing it.

File: pkg/controller/restapi/operation/vdri/operation.go (hunk 2 above)
Comment (sudeshrshetty) [reply pkg/controller/restapi/operation/vdri/operation.go]: fixed

File: pkg/controller/restapi/operation/messaging/operation.go (hunk 1 above)
Comment (sudeshrshetty) [reply pkg/controller/restapi/operation/messaging/operation.go]: fixed

File: pkg/controller/restapi/restapi.go [hunk 4]
```diff
@@ -1,25 +1,29 @@
 /*
-Copyright SecureKey Technologies Inc. All Rights Reserved.
-
-SPDX-License-Identifier: Apache-2.0
-*/
+ *
```
Comment (troyronda) [inline pkg/controller/restapi/restapi.go]: should revert this change.

File: pkg/controller/restapi/restapi_test.go (hunk 3 above)
Comment (troyronda) [reply pkg/controller/restapi/restapi_test.go]: should revert this change.

Comment (sudeshrshetty) [reply pkg/controller/restapi/restapi_test.go]: reverted
